From: Kshitij Kulshreshtha <kshitij@math.upb.de>
Subject: increment the correct pointer in log_op reverse mode

this op has no use for Targ2 but Targ is used in the statements above
so incrementing Targ2 is the silliest mistake I've seen so far.

Bug-Debian: https://bugs.debian.org/771222
Origin: upstream, https://projects.coin-or.org/ADOL-C/changeset/568
--- a/ADOL-C/src/ho_rev.c
+++ b/ADOL-C/src/ho_rev.c
@@ -1517,7 +1517,7 @@
 
                       HOV_INC(Ares, k)
                       HOV_INC(Aarg, k)
-                      HOS_OV_INC(Targ2, k)
+                      HOS_OV_INC(Targ, k)
                   }
             }
                 break;
